src/contexts/help.context.tsx

```diff
@@ -0,0 +1,23 @@
+import { createContext } from 'react';
+import { Step } from 'react-joyride';
+
+let helpSteps: Step[] = [];
+
+let getHelpSteps = (): Step[] => {
+    const firstStep = helpSteps[0];
+    //Below line is to prevent application breaking if element is not present for any reason (e.g. if the user deletes build or if there is no data.)
+    if (firstStep && document.getElementById(firstStep.target.toString().slice(1))) {
+        helpSteps.every((e) => {
```

Comment:
not sure if this is working as expected
`Array.prototype.every() expects a return value from arrow function  array-callback-return`

Reply:
Thanks. Changed it to forEach()